app/poros/merchant.rb

```diff
@@ -0,0 +1,9 @@
+class Merchant
+
+  attr_reader :name, :id
+
+  def initialize(merchant_data)
+    @name = merchant_data[:attributes][:name]
+    @id = merchant_data[:id].to_i
+  end
+end
```

Comment:
Careful naming attributes "id", since POROs automatically get a unique id assigned to them. This could cause some conflicts later on.